### PR TITLE
Update Russian translate

### PR DESCRIPTION
**Pull request by deniko1985**
I translated additional settings DerpFest

### CODE REVIEW

**Review by idoybh — APPROVED** [review]

File: res/values-ru/derpquest_strings.xml [hunk 1]
```diff
@@ -373,6 +402,7 @@
     <string name="status_bar_battery_style_circle">Заполненый круг</string>
     <string name="status_bar_battery_style_text">Текст</string>
     <string name="status_bar_battery_style_hidden">Спрятан</string>
+    <string name="battery_percent_title">Проценты</string>
```
Comment (idoybh) [inline res/values-ru/derpquest_strings.xml:400]: defined twice

File: res/values-ru/derpquest_strings.xml [hunk 2]
```diff
@@ -544,11 +578,30 @@
...
+    <string name="show_gentle_header_title">Показать заголовки уведомлений</string>
+    <string name="show_gentle_header_summary">Показывать ли заголовок «тихие уведомления» над такими уведомлениями (требуется перезагрузка интерфейса системы)</string>
+
     <!-- flash on call waiting -->
     <string name="flash_on_call_category">Мигающая вспышка</string>
     <string name="flash_on_call_title">Мигающая вспышка при входящем вызове</string>
     <string name="flash_on_call_delay_title">Частота мигания вспышки</string>
 
+ <!-- Statusbar Clock Font Style, Size & Color -->
+    <string name="status_bar_clock_font_style_title">Стиль шрифта часов</string>
```
Comment (idoybh) [inline res/values-ru/derpquest_strings.xml:586]: defined twice

File: res/values-ru/derpquest_strings.xml [hunk 3]
```diff
@@ -544,11 +578,30 @@
...
+    <string name="status_bar_clock_font_size_title">Размер шрифта часов</string>
```
Comment (idoybh) [inline res/values-ru/derpquest_strings.xml:587]: sames

File: res/values-ru/derpquest_strings.xml [hunk 4]
```diff
@@ -745,4 +794,85 @@
...
+    <string name="theme_select_title">Темы</string>
+    <string name="theme_select_summary">Просмотрите и выберите установленные темы</string>
+
+    <!--Gestures-->
+    <string name="aosp_gesture_title">AOSP жесты</string>
+
+    <!-- Units -->
+    <string name="unit_seconds">секунды</string>
+    <string name="unit_milliseconds">милисекунды</string>
+    <string name="unit_pixels">dp</string>
```
Comment (idoybh) [inline res/values-ru/derpquest_strings.xml]: same

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by idoybh** [comment]
Force push to the same branch with the fixes please. Spasiva!

File: res/values-ru/derpquest_strings.xml [hunk 5]
```diff
@@ -253,6 +279,9 @@
     <string name="clock_date_left">Слева от часов</string>
     <string name="clock_date_position">Расположение даты</string>
 
+<!-- Lockscreen information -->
+    <string name="lockscreen_info_category_title">Информационные элементы экрана блокировки</string>
+
     <!-- Statusbar Clock Font Style & Size -->
     <string name="status_bar_clock_font_style_title">Стиль шрифта часов</string>
```
Comment (idoybh) [inline res/values-ru/derpquest_strings.xml]:
```suggestion

```

Comment (idoybh) [reply res/values-ru/derpquest_strings.xml]: for example

**Review by idoybh — CHANGES_REQUESTED** [review]

**Comment by deniko1985** [comment]
Update

**Review by idoybh — APPROVED** [review]

**Comment by idoybh** [comment]
Thank you kindly

**Pull request merged**